build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -171,38 +171,11 @@
     <CodeAnalysisRules Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
     <CodeAnalysisRuleAssemblies Condition="'$(Configuration)|$(Platform)'=='Release|x64'" />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='ReleaseWithoutAsm|Win32'">
```

Comment:
Mmm... AFAIK, the "vc14" folder ( visual studio 2015 projects) are not part of the official sources ( https://sourceforge.net/projects/libpng/files/zlib/1.2.8/ ). And if we compare the file with the older vc11, it does not contain this part.
So, now... What am I supposed to do?